client/src/actions/users.js

```diff
@@ -24,7 +24,7 @@ export const FETCH_CURRENT_USER_FULFILLED = 'FETCH_CURRENT_USER_FULFILLED';
 
 export const createUser = createAction(CREATE_USER, userService.create);
 export const fetchCurrentUser = createAction(FETCH_CURRENT_USER, userService.fetchSelf);
-export const fetchUsers = createAction(FETCH_USER, userService.fetchAll);
+export const fetchUsers = createAction(FETCH_USERS_FULFILLED, userService.fetchAll);
```

Comment:
It only defines actions. Fulfillment of promise is a criteria which do not have any concerns with functions call